pkgs/by-name/ew/eww/package.nix

```diff
@@ -32,18 +46,27 @@ rustPlatform.buildRustPackage rec {
     librsvg
   ];
 
-  cargoBuildFlags = [ "--bin" "eww" ];
+  cargoBuildFlags = [
+    "--bin"
+    "eww"
+  ];
 
   cargoTestFlags = cargoBuildFlags;
 
   # requires unstable rust features
   RUSTC_BOOTSTRAP = 1;
 
   meta = {
-    description = "ElKowars wacky widgets";
+    description = "Elkowars Wacky Widgets, a widget system made in Rust to allow you create widgets for any WM.";
```

Comment:
This can go in longDescription but not description, this is a bit long and doesn't follow guidelines (shouldn't end in period). Alternatively you can rephrase this to be shorter

Reply:
Rephrased the description to be shorter
Moved the long stuff to longDescription, does this look fine now?